getting-started/templates/systemlink-values.yaml

```diff
@@ -699,3 +699,17 @@ license:
     ## Leave this value as empty, to use the default storage class, if it already supports ReadWriteMany access.
     ##
     storageClassName: ""
+
+## Configuration for event-based routine triggering.
```

Comment:
Do we expect users to need to adjust these settings. Ideally this document just calls out configuration options that are likely to need user attention. User's can still adjust settings that are not declared here.

Reply:
We already set reasonable values in the top-level helm chart based on our current calculations: https://ni.visualstudio.com/DevCentral/_git/Skyline?path=/Enterprise/SystemLink/helm/values.yaml&version=GBusers/gpappsze/include-new-routine-related-charts-in-bundle&line=675&lineEnd=687&lineStartColumn=1&lineEndColumn=18&lineStyle=plain&_a=contents

So this is just for documentation purpose. It's likely that eventually clients will need to set these as they scale up to bigger workloads

Reply:
Can we provide suggestions on how to scale these values in the documentation. Should we add an `<ATTENTION>` tag to call attention to these settings?

Reply:
hmmm... based on this I'm leaning towards not documenting this:
> User's can still adjust settings that are not declared here.

Especially since we don't have a guide on how to scale this service